src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -749,7 +750,7 @@ private function deleteAuditLogs(Document $project, callable $getProjectDB, stri
      * @return void
      * @throws Exception
      */
-    private function deleteSite(Database $dbForPlatform, callable $getProjectDB, Device $deviceForSites, Device $deviceForFunctions, Device $deviceForBuilds, Document $document, CertificatesAdapter $certificates, Document $project): void
+    private function deleteSite(Database $dbForPlatform, callable $getProjectDB, Device $deviceForSites, Device $deviceForFunctions, Device $deviceForBuilds, Device $deviceForFiles, Document $document, CertificatesAdapter $certificates, Document $project): void
```

Comment:
Do we also need to do this when a single deployment is deleted?

Reply:
Yes, nice catch